packages/umi-rpc-web3js/src/createWeb3JsRpc.ts

```diff
@@ -123,7 +126,7 @@ export function createWeb3JsRpc(
       toWeb3JsPublicKey(programId),
       {
         ...options,
-        filters: options.filters?.map((filter) => parseDataFilter(filter)),
+        filters: options.filters?.map((filter: any) => parseDataFilter(filter)),
```

Comment:
wondering what the reason why this needs to be casted?

Reply:
I think my TS was just complaining about it so I filled it in. I'll double check that one and come back.

Reply:
Just double checked and my TS is not complaining. No idea why that has been added or for what reason, seems weird for me to add that. 

I'll remove it in the next commit.